src/cnc/flavors/aws/ecs/1/build/build_lambda_functions.sh.j2

```diff
@@ -0,0 +1,54 @@
+send_{{ service.name }}_status_hook () {
```

Comment:
We should remove this status hook function. It's already defined in a parent template.